appservice/src/intTest/groovy/nu/yona/server/AddDeviceTest.groovy

```diff
@@ -0,0 +1,145 @@
+package nu.yona.server
+
+import groovyx.net.http.RESTClient
+import groovyx.net.http.HttpResponseException
+import spock.lang.Ignore
+import spock.lang.IgnoreRest
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+import groovy.json.*
+
+class AddDeviceTest extends Specification {
+
+	def appServiceBaseURL = System.properties.'yona.appservice.url'
+	def YonaServer appService = new YonaServer(appServiceBaseURL)
+	
+	@Shared
+	def richardQuinURL
+	@Shared
+	def richardQuinPassword = "R i c h a r d"
+	@Shared
+	def newDeviceRequestExpirationDateTime
+	
+	def 'Add user Richard Quin'(){
+		given:
+
+		when:
+			def response = appService.addUser("""{
+				"firstName":"Richard",
+				"lastName":"Quin",
+				"nickName":"RQ",
+				"emailAddress":"rich@quin.net",
+				"mobileNumber":"+12345678",
```

Comment:
Set the mobile number as a timestamp. For debuggability, it's good to add that to the first and last name too.